Migration.Toolkit.Core/Handlers/MigrateUsersCommandHandler.cs

```diff
@@ -65,8 +65,9 @@ public async Task<CommandResult> Handle(MigrateUsersCommand request, Cancellatio
         await MigrateCmsRoles(kx13Context, cancellationToken, migratedSiteIds);
 
         var kx13CmsUsers = kx13Context.CmsUsers
-                .Include(u => u.CmsUserRoles.Where(x => migratedSiteIds.Contains(x.Role.SiteId!.Value) || x.Role.SiteId == null))
-                .ThenInclude(ur => ur.Role)
+                .Where(u => new[] { 2, 3 }.Contains(u.UserPrivilegeLevel))
```

Comment:
Please check whether exists suitable enum for privileges and use it. Otherwise define two constants for these values.